### ISSUE TITLE
question mark on quaffing and eating seems broken

### ISSUE THREAD

**Issue by runlow** [Elronnd/lrogue#4]
**Expected:**
When picking up a potion then pressing `q` and `?` it should show the potions in the inventory.
Same thing goes for eating, zapping, wearing and throwing.

**Observed:**
It shows nothing or it shows a list of scrolls instead of potions.

**Steps:**
Eating:
1. Start the game
2. Press `e` then `?`
3. Notice that your food is not listed. Same goes for any food you find.

Quaffing:
1. Find a potion in the dungeon.
2. Press `q` then `?`
3. Notice that the potion is not listed

Listing scrolls seems to work as expected.

**Comment by runlow** [comment]
Same thing goes for wands. Pressing `?` upon pressing `z` lists scrolls, not wands.

**Comment by moon-chilled** [comment]
`?` is the symbol for a scroll, so it shows you scrolls.  If you want to see potions, use `!`; `:` for food; etc.

**Comment by runlow** [comment]
Seems you're right. Strange I didn't realise it. Thanks!

The FreeBSD version and the Debian `freebsd-nonfree` package rogue would list relevant items anyway when pressing `?` which made me think `?` is the key to press to list things.

Upon checking again now it seems those versions (FreeBSD and Debian package) would list *relevant* things even if you pressed `*` or `/` etc and not just `?`. Unlike `lrogue2` they would only show *relevant* items. I'm not saying that this behaviour here is wrong or worse (having to press the relevant key). Just had a habit of always pressing `?` which was wrong to begin with so thought this was broken. :smile:

In other words on those systems if I wanted to eat I would press `e` then `?` and it would list food.
On `lrogue2` it's `e` then `:`. Just pointing it out.
